ar_lazy_preload.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = ">= 2.3"
 
-  s.add_dependency "rails", ">= 4.2"
+  s.add_dependency "rails", ">= 6.0"
```

Comment:
We'll have to keep `>= 4.2` to keep the backward compatibility

Reply:
Didn't you drop support for rails 4.2? https://github.com/DmitryTsepelev/ar_lazy_preload/pull/45/files.

Reply:
Good point, but we should keep >= 5.0 for now 🙂https://github.com/DmitryTsepelev/ar_lazy_preload/pull/45/files#diff-635b20966b1d2fea76ee0565254eab939d55f6d22e3c4727cf9d453f589b87a0R3